system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -612,7 +612,7 @@ impl pallet_proxy::Config for Runtime {
 	type CallHasher = BlakeTwo256;
 	type AnnouncementDepositBase = AnnouncementDepositBase;
 	type AnnouncementDepositFactor = AnnouncementDepositFactor;
-	type BlockNumberProvider = System;
+	type BlockNumberProvider = RelaychainDataProvider<Runtime>;
```

Comment:
We assume that nobody used this already? Otherwise the proxy delay would be wrong.

Reply:
yes. I mentioned this in PR description. there is no use currently. and even if someone would before we release. I believe the impact is low. the RC blocks are higher than AH blocks.

Reply:
Okay we will need to still ensure that wallets & integrations are aware of this.  

cc @anaelleltd @SBalaguer   
Proxy pallet on Polkadot Asset Hub will use the Relay block chain numbers to measure delays instead of Asset Hub block number. This will introduce some friction for dApps, but it is needed since AH block times are drifting too much from real world time.

Reply:
yes, I already talked to @anaelleltd. We waiting for this PR to be approved.